tests/unit_tests/test_sin.py

```diff
@@ -0,0 +1,53 @@
+import json
+import math
+
+import pytest
+
+from tests.utils import load_process_code, run_process
+
+
+@pytest.fixture
+def sin_process_code():
+    return load_process_code("sin")
+
+
+@pytest.mark.parametrize(
+    "example_input,expected_output",
+    [
+        ({"x": 0}, 0),  # sin(0°)
+        ({"x": math.pi / 6}, pytest.approx(0.5)),  # sin(30°)
```

Comment:
Typically we use this directly in the assert statement, see e.g. `test_multiply` - just so you only need to write it once. But it's fine to use it like that as well, to signal explicitly that you don't expect an exact answer.

Reply:
Thanks, I changed this because it's easier to add tests if needed.